toolkit/tools/imagegen/configuration/systemconfig_test.go

```diff
@@ -52,14 +52,37 @@ func TestShouldFailParsingMissingPackages_SystemConfig(t *testing.T) {
 
 	missingPackageListConfig := validSystemConfig
 	missingPackageListConfig.PackageLists = []string{}
+	missingPackageListConfig.Packages = []string{}
 
 	err := missingPackageListConfig.IsValid()
 	assert.Error(t, err)
-	assert.Equal(t, "system configuration must provide at least one package list inside the [PackageLists] field", err.Error())
+	assert.Equal(t, "system configuration must provide at least one package list inside the [PackageLists] or one package in the [Packages] field", err.Error())
 
 	err = remarshalJSON(missingPackageListConfig, &checkedSystemConfig)
 	assert.Error(t, err)
-	assert.Equal(t, "failed to parse [SystemConfig]: system configuration must provide at least one package list inside the [PackageLists] field", err.Error())
+	assert.Equal(t, "failed to parse [SystemConfig]: system configuration must provide at least one package list inside the [PackageLists] or one package in the [Packages] field", err.Error())
+}
+
+func TestShouldSucceedParsingMissingPackages_SystemConfig(t *testing.T) {
```

Comment:
Looks good, I'd only suggest splitting this to do one scenario per test. This way the tests are more precise and the test name can quickly guide you to what might be wrong in case of a failure.